gui/panel_brainentropy.m

```diff
@@ -1706,7 +1701,7 @@ function import_baseline( Lst,Frmt  )
     global MEMglobal
 
     MEMpaneloptions.InverseMethod           = 'MEM';
-    MEMpaneloptions.automatic.MEMexpert     =   strcmp( char(ctrl.jButEXP.getText()),'Expert' );                    
+    MEMpaneloptions.automatic.MEMexpert     =   OPTIONS.automatic.MEMexpert;              
```

Comment:
This line doesnt work: 

MEMpaneloptions.automatic.MEMexpert     =   OPTIONS.automatic.MEMexpert;              
Unable to resolve the name 'OPTIONS.automatic.MEMexpert'.

Reply:
Fixed in https://github.com/Edouard2laire/best-brainstorm/pull/5/commits/1a959d34a9a7ac20bf1626c677c7bffe2680c7a0

@ilianAZZ is this fix ok for you?